### ISSUE TITLE
USWDS - Bug: [Mobile Banner Focus Outline Hard To Discern]

### ISSUE THREAD

**Issue by jhancock532** [uswds/uswds#5012]
### Describe the bug

The focus outline for the mobile gov banner is currently rendering outside the page viewport on three sides.

### Steps to reproduce the bug

1. Go to an implementation of the gov banner component, e.g. [cloud.gov](https://cloud.gov/)
2. Resize the window to mobile dimensions.
3. Tab to the banner.

### Expected Behavior

This focus indicator should be clearly visible on all sides of the banner. 

Rendering only an underline border on focus causes confusion to users with vision difficulties who have scaled the page to 400% zoom.

### Related code

This can be fixed by adding a CSS style rule, `outline-offset: -4px;`, which brings the focus indicator within the component on all sides. This style rule should only apply to mobile styles, as on tablet the focus indicator for this component surrounds a link instead.

### Screenshots

**Component without focus:**

<img width="384" alt="Screenshot 2022-11-03 at 12 30 38" src="https://user-images.githubusercontent.com/18164832/199721249-4dadabd8-310a-4f97-ac9e-77071dbab2e3.png">

**Component with focus:**

<img width="385" alt="Screenshot 2022-11-03 at 12 31 01" src="https://user-images.githubusercontent.com/18164832/199721288-488ae25f-ce81-4b3f-a3ba-decef173e6f4.png">

**Proposed solution style:**

<img width="388" alt="image" src="https://user-images.githubusercontent.com/18164832/199735622-a5d00b06-3bb0-4a57-8fb6-130e3fe454c8.png">

### System setup

- USWDS 3.1
- MacBook Pro
- MacOS Monterey 12.3.1
- Chrome 107.0.5304.87

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by thisisdano** [comment]
Fixed by https://github.com/uswds/uswds/pull/5013
